pkg/agent/apiserver/handlers/agentinfo/handler.go

```diff
@@ -60,3 +62,43 @@ func HandleFunc(aq monitor.AgentQuerier) http.HandlerFunc {
 		}
 	}
 }
+
+var _ common.TableOutput = new(AntreaAgentInfoResponse)
+
+func (r AntreaAgentInfoResponse) GetTableHeader() []string {
+	return []string{"VERSION", "POD", "NODE", "NODE-SUBNET", "OVS-VERSION", "OVS-BRIDGE-NAME", "OVS-FLOW-TABLE", "NETWORK-POLICIES", "ADDRESS-GROUPS", "APPLIED-TO-GROUPS", "LOCAL-PODS", "AGENT-CONDITIONS"}
```

Comment:
Thanks for making this change. I tried the command, the line is a little long, Generally the table output should be an informative summary, not having to list all content, so I think we can remove some fields, like this:
```
POD                                            NODE      STATUS  NETWORK-POLICIES ADDRESS-GROUPS APPLIED-TO-GROUPS CONNECTED-AGENTS
kube-system/antrea-controller-7cffdf584b-s276j dev4linux Healthy 0                0              0                 2  
```
The status is got from conditions, when all conditions are true, it's healthy, otherwise unhealthy.

Reply:
Done.

Reply:
Done.